yt_dlp/extractor/reverbnation.py

```diff
@@ -1,18 +1,20 @@
+import functools
+
 from .common import InfoExtractor
-from ..utils import (
-    qualities,
-    str_or_none,
-)
+from ..utils import InAdvancePagedList, int_or_none, qualities, str_or_none, traverse_obj
```

Comment:
```suggestion
from ..utils import (
    InAdvancePagedList,
    int_or_none,
    qualities,
    str_or_none,
    traverse_obj,
)
```

Reply:
This is the default formatting, so cannot change it.